presto-product-tests/conf/tempto/tempto-configuration-for-docker-ldap.yaml

```diff
@@ -0,0 +1,28 @@
+databases:
+  hive:
+    host: hadoop-master
+  presto:
+    host: presto-master
+    configured_hdfs_user: hive
+    cli_ldap_authentication: true
+    cli_ldap_truststore_path: /etc/ldap/cacerts.jks
+    cli_ldap_truststore_password: testldap
+    cli_ldap_user_name: financeuser1
+    cli_ldap_user_password: TCAMPass123
+    cli_ldap_server_address: https://${databases.presto.host}:8443
+
+  mysql:
```

Comment:
We add those sections (`mysql`, `postgree`) in every single profile? They are always the same. Can't we just move it to the base tempto configuration which is located in product tests resources? If so please do such refactor in a separate commit, so we can push it to master futher without pushing other changes from this patch.